clients/js/src/examples/multisig.ts

```diff
@@ -153,14 +178,13 @@ async function main() {
   });
   const signedWrapTxB = await partiallySignTransactionMessageWithSigners(wrapTxB);
 
```

Comment:
With multisig helpers, it did not feel correct converting these to return instructions as the standard workflow is designed to share pre-signed tx's and combine them.

Reply:
Given the amount of variables required, they definitely feel a bit wieldy. However, I find that it's sort of the nature of a multisig transaction. If you look at the [CLI example](https://www.solana-program.com/docs/token-wrap#wrap-tokens-spl-token-multisig) it's similar though. For that reason, I think we are kinda at the frontier for API improvements for it for now 🤔